topologies/datacenter-latest/labguides/source/bgp-with-cvp.rst

```diff
@@ -0,0 +1,170 @@
+
+   2. Notice that BGP does not appear to be configured on leaf4.
+
+   3. Switch to **spine1** to see the status of spine1's BGP configuration.
+
+    .. image:: images/cvp-l3ls/spine1-bgp-overview-pre.png
+       :align: center
+       :width: 50 %
+
+   3. See that there is 1 unestablished peer and we can see the details for that attempted neighborship in the table.
+
+   4. View these details for **spine2** as well.
+
+3. Click **Metrics** at the top of the page
+
+   1. In this section of CloudVision, users can create custom Dashboards to refer to particular telemetry data they find noteworthy.
+
+   2. Click **create a new dashboard**.
+
+   3. In the view builder on the left, select the values for each dropdown as listed below:
+    
+    ..   .. table::
+       :widths: auto
+       :align: center
+
+       ==============  =========================
+       Dashboard View
+       -----------------------------------------                         
+         View Mode     Table
+        Metric Type    Devices                
+          Metrics      BGP                     
+                        - Established Peers    
+                        - Unestablished Peers  
+                        - Learned Paths        
+                        - AS Number            
+                        - Router-ID            
+          Devices       - leaf1                 
+                        - leaf2                 
+                        - leaf3                 
+                        - leaf4                 
+                        - spine1                
+                        - spine2                
+       ==============  =========================
+
+    .. image:: images/cvp-l3ls/bgp-dashboard-setup.png
+       :align: center
+       :width: 50 %
+
+   4. Click **Save Dashboard** in the bottom left corner.
+   
+   5. If prompted to name the dashboard, give a name and click **Save**.
+
+   6. Now there is a dashboard that displays BGP information for all switches in our leaf-spine network in one place.
+
+4. Configure BGP on leaf4.
+
+   1. Click **Provisioning** at the top of the page.
+
+   2. Find **leaf4**, right click on it, and click **Manage -> Configlet**.
+
+    .. image:: images/cvp-l3ls/leaf4-manage-configlet.png
+       :align: center
+       :width: 50 %
+
+   3. Search for ``Leaf4-BGP-Lab-Full`` in the search bar, select the configlet, and click **Validate**.
+
+    .. image:: images/cvp-l3ls/leaf4-add-bgp-configlet.png
+       :align: center
+       :width: 50 %
+
+   4. Validate the Designed Configuration created by CloudVision from the Proposed Configlets against Leaf4's running configuration and click **Save**.
+
+    .. image:: images/cvp-l3ls/leaf4-validate-bgp-configlet.png
+       :align: center
+       :width: 50 % 
+
+   5.  There should now be a temporary action for leaf4 indicated by the green outline around leaf4. Click **Save**.
+
+    .. image:: images/cvp-l3ls/leaf4-temp-action.png
```

Comment:
@jrecchia1029 update this screenshot as it references a container structure for the Leaf switches that won't exist.